src/components/pages/career-details/career-form/career-components.tsx

```diff
@@ -109,12 +109,14 @@ export const Actions = ({
     return (
       <>
         <FormError error={error} />
-        <Trans i18nKey={'career.again-text'}>
+        <br />
+        <Trans i18nKey={'career.action.again-text'}>
           Versuch es bitte noch einmal. Klappt es nicht dann schicke deine
           Bewerbung direkt an <strong>career@satellytes.com</strong>
         </Trans>
+        <br />
         <SendButton onClick={() => tryAgainFn()}>
```

Comment:
this fixes the layout a little bit at least when an error is displayed.
![Screen Shot 2022-02-04 at 16 02 03](https://user-images.githubusercontent.com/1701755/152551546-cd00c077-cdc7-4d5b-a298-0a4f1fe9b20a.png)